docs/book/src/component-config-tutorial/testdata/project/config/webhook/service.yaml

```diff
@@ -1,12 +0,0 @@
-
-apiVersion: v1
-kind: Service
-metadata:
```

Comment:
we need to re-added the missing files.

Reply:
There is no web hook logic for this operator, why is this required?

Reply:
Fair enough